randovania/gui/ui_files/main_window.ui

```diff
@@ -653,6 +661,28 @@ For more information, check [Randovania Help](help://tab_multiworld)!</string>
     <string>Verify installation</string>
    </property>
   </action>
+  <action name="menu_action_sound_alerts">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="checked">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Sound Alerts</string>
+   </property>
+  </action>
+  <action name="menu_action_visual_taskbar_dock_alerts">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="checked">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Visual Taskbar/Dock Alerts</string>
```

Comment:
I would just say `Visual Alerts`

Reply:
My concern with just "Visual Alerts" is that user can't really tell what it's supposed to affect. Saying Taskbar/Dock would at least give them an indication of that.

Reply:
The issue is that this is a *very* OS specific thing. So technically this would be more "Taskbar/Dock/Panel/topbar/whatever-android-has/whatever-ios-has" which is just a complete mess
IMO if the user doesnt need to know what exact audio alerts they are, they dont need to know the exact visual alerts.

Reply:
Changed it to
![image](https://github.com/user-attachments/assets/fb8ee42b-598f-415c-9339-ab6a822c1256)